sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1749,6 +1749,10 @@ public void updateCompleteness(Case caze) {
 	 */
 	public void updateVaccinationStatuses(Long personId, Disease disease, Date vaccinationDate) {
 
+		if (vaccinationDate != null) {
```

Comment:
Add a comment: "Vaccination only counts when it's on the day before".. or something like that.

Reply:
Also you have to change the greaterThanOrEqualTo to a greaterThan in the datePredicate below